packages/flutter/lib/src/rendering/table_border.dart

```diff
@@ -74,6 +74,8 @@ class TableBorder {
   final BorderSide verticalInside;
 
   /// The [BorderRadius] to use when painting the corners of this border.
+  ///
+  /// It is also applied to [DataTable]'s Material for clipping.
```

Comment:
Breadcrumb nit

```suggestion
  /// It is also applied to [DataTable]'s [Material] for clipping.
```

Does the borderRadius cause the clip?

Reply:
`borderRadius` is simply supplied to `Material`. So the `Material` has the same `borderRadius` as the `Table`.

Reply:
Material's `ClipBehavior` is the default (`Clip.none` ) here so I updated the comment to just: 

>   /// It is also applied to [DataTable]'s [Material].